src/features/ModelPlan/ReadOnly/_components/FilterView/util.tsx

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
 
+export const filterGroupParams = [
```

Comment:
Can this section reference the GQL type instead? (Or is it not possible because of casing or some other reason?)

Reply:
Thanks Steven, yeah they need to be converted to lowercase to key of the url routes.  I went ahead and did that mapping